apps/datafeeder/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { TestBed, async } from '@angular/core/testing'
+import { RouterTestingModule } from '@angular/router/testing'
+import { AppComponent } from './app.component'
+
+describe('AppComponent', () => {
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [RouterTestingModule],
+      declarations: [AppComponent],
+    }).compileComponents()
+  }))
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent)
+    const app = fixture.componentInstance
+    expect(app).toBeTruthy()
+  })
+
+  it(`should have as title 'datafeeder'`, () => {
+    const fixture = TestBed.createComponent(AppComponent)
+    const app = fixture.componentInstance
+    expect(app.title).toEqual('datafeeder')
+  })
+
+  it('should render title', () => {
+    const fixture = TestBed.createComponent(AppComponent)
```

Comment:
wonder how it passes as i see nothing in the template

Reply:
It seems test was skipped because of miss test target in agular.json